packaging/windows/clisdk/bundle.wxs

```diff
@@ -32,13 +34,13 @@
     <util:DirectorySearch Path="[PreviousInstallFolder]" Variable="DOTNETHOME" After="PreviousInstallFolderSearch" Condition="PreviousInstallFolder" />
 
     <Chain DisableSystemRestore="yes" ParallelCache="yes">
-      <MsiPackage SourceFile="$(var.CLISDKMsiSourcePath)">
+      <MsiPackage SourceFile="$(var.CLISDKMsiSourcePath)" Visible="yes">
```

Comment:
The only way this will work is if we ensure that every other bundle that ships these MSIs also have them marked as visible, otherwise a bundle that carries a newer version of these MSIs and have them hidden will end up removing them from ARP.